src/components/Login/ForgotPassword.jsx

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
Yes, I agree; since we have already used `htmlFor` here but still got this error. I think the problems come from the reactstrap; the `id` didn't work on its components.